pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -422,6 +482,12 @@ class _MCPServerHTTP(MCPServer):
     allow_sampling: bool = True
     """Whether to allow MCP sampling through this client."""
 
+    max_retries: int = 1
```

Comment:
I feel like we should consider supporting a `tenacity.Retrying` here. I guess that can come later.